src/component/Drop/Drop.js

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import IconButton from "@mui/material/IconButton";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import Button from "@mui/material/Button";
+import HdrWeakIcon from "@mui/icons-material/HdrWeak";
+import Divider from "@mui/material/Divider";
+
+import "./Drop.css";
+const options = [
+  "Development",
+  "Design",
+  "Marketing",
+  "Music",
+  "Life Style",
+  "IT & Software",
+  "Personal Development",
+  "Health & Fitness",
+  "Teaching",
+  "Math & Logic",
+  <Divider />,
+  <Button className="btn">View all Categoryies</Button>,
+];
+
+const ITEM_HEIGHT = 48;
+
+export default function LongMenu() {

```

Comment:
It's either LogMenu or Drop
Decide on one name for the exported function and the file's